template/catppuccin.user.css

```diff
@@ -77,29 +70,29 @@
     }
   }
 
-  /*  Application of the Theme
-  *  The values below are examples and may need to be adapted.
-  *  To apply the correct Theming call the above defined Mixin and pass
-  *  your desired Flavor and Accent Color to it. These are usually
-  *  populated through the @vars in the Meta Block. */
-  // synced colorscheme
+  /* This section applies the light/dark themes automatically based on the user's system theme.
+     Uncomment this *only* if the website you are theming doesn't have a light/dark mode toggle.
   @media (prefers-color-scheme: light) {
-    .no-theme {
+    :root {
       #catppuccin(@lightFlavor, @accentColor);
     }
   }
   @media (prefers-color-scheme: dark) {
-    .no-theme {
+    :root {
       #catppuccin(@darkFlavor, @accentColor);
     }
   }
+  */
 
-  // manually set colorscheme
-  .light-theme {
+  /* If the website you are theming *does* have a light/dark mode toggle,
+     you can conditionally enable light/dark flavors by targeting whatever attribute that website uses to indicate the theme.
+     Un-comment this block if that is the case.
+  :root[data-theme="dark"] {
```

Comment:
I think leaving a reference to https://github.com/catppuccin/userstyles/blob/39288b593278e5efa48007d5fc001292cf910672/styles/youtube/catppuccin.user.css#L21-L30

And perhaps a reference to https://github.com/catppuccin/userstyles/blob/39288b593278e5efa48007d5fc001292cf910672/styles/brave-search/catppuccin.user.css#L16-L29

Or these could be placed in the `how-to-write-a-userstyle.md` file.

Reply:
Ohh yeah smart, I'll do that.